graphistry/dgl_utils.py

```diff
@@ -18,27 +19,26 @@
     XSymbolic,
     YSymbolic,
     resolve_X,
-    resolve_y
+    resolve_y,
 )
 from .util import setup_logger
 
-logger = setup_logger(__name__)
+logger = setup_logger(name=__name__, verbose=config.VERBOSE)
```

Comment:
verbose=config.VERBOSE) is copy-pasted in almost all invocations, so maybe better to do:

`def setup_logger(..., verbose=config.VERBOSE)`